behave_django/management/commands/behave.py

```diff
@@ -54,6 +55,14 @@ def add_command_arguments(parser):
         help="Use simple test runner that supports Django's"
         " testing client only (no web browser automation)"
     )
+    parser.add_argument(
+        '--runner-class',
+        action='store',
+        type=valid_python_module,
+        help=('Full Python dotted path to a package, module, Django '
+              'TestRunner.  Defaults to '
+              '"behave_django.runner.BehaviorDrivenTestRunner".')
```

Comment:
It's a bit sad that we can't simply set `behave_django.runner.BehaviorDrivenTestRunner` as a `default` attribute in this parser argument (because we have it [hard-coded in the management command](https://github.com/behave/behave-django/blob/main/behave_django/management/commands/behave.py#L144)). I feel that would make things very elegant and transparent. :worried:

Reply:
I played with this exact idea, and turned it down, because of how the (`simple`/`use-existing`) flags work. It could be worth exploring in the future to simply disable those flags and explicitly state you want them by doing: `--runner-class behave_django.runner.Simple.....`

But that can be down the line